internal/config/constants.go

```diff
@@ -21,7 +21,7 @@ const (
 
 	// BuildKitContainerVersion is the version of buildkit container image.
 	// renovate: datasource=docker versioning=docker depName=moby/buildkit
-	BuildKitContainerVersion = "v0.15.0"
+	BuildKitContainerVersion = "v0.15.1"
```

Comment:
`make rekres` is missing

Reply:
Nope. No occurence of `moby/buildkit` in the kres module, so no update required. Running `make kres && ./_out/kres-linux-amd64 gen` also gives nothing.

Reply:
ah, ok... I guess we're using default from the CI? @frezbo should we bump there as well?

Reply:
yeh, let's update in https://github.com/siderolabs/build-container/ too

Reply:
Done in https://github.com/siderolabs/build-container/pull/118